crates/wysiwyg/src/dom/nodes/container_node.rs

```diff
@@ -149,6 +178,22 @@ impl ContainerNode<u16> {
             handle: DomHandle::new_unset(),
         }
     }
+
+    pub fn is_empty_list_item(&self) -> bool {
+        match self.kind {
+            ContainerNodeKind::ListItem() => {
+                let html = self.to_html();
+                if html == "<li></li>".to_html()
+                    || html == "<li>\u{200b}</li>".to_html()
```

Comment:
This `if` could be `return html == "<li></li>".to_html() || html == "<li>\u{200b}</li>".to_html()`